RESTService/pyminsky.cc

```diff
@@ -30,28 +30,73 @@ namespace pyminsky
 
   minsky::Item& findObject(const std::string& typeName)
   {
-    using namespace minsky;
-    auto& canvas = minsky.canvas;
+      using namespace minsky;
+      auto& canvas = minsky.canvas;
 
-    canvas.item.reset();
+      canvas.item.reset(); // Reset the item to ensure no leftover references
```

Comment:
indentation has gone a bit weird

Reply:
Oops. I must have accidentally pressed the tab key on that line. I'll check the indentation more carefully prior to a pull request next time.